server/src/main/java/com/linecorp/centraldogma/server/CentralDogma.java

```diff
@@ -681,6 +684,13 @@ protected HttpResponse doGet(ServiceRequestContext ctx, HttpRequest req) {
         // Enable content compression for API responses.
         decorator = decorator.andThen(contentEncodingDecorator());
 
+        // Config cors policy
+        Function<? super HttpService, CorsService> corsDecorator =
+                configCorsDecorator(config().corsAllowedOrigins());
+        if (corsDecorator != null) {
+            decorator = decorator.andThen(corsDecorator);
+        }
```

Comment:
How about taking `decorator` as a parameter of `configCorsDecorator()` method?
```
maybeConfigCors(config().corsAllowedOrigins(), decorator);
```

Reply:
i don't think that would work, `andThen` returns a new function, not altering the input function